src/constants/constants.ts

```diff
@@ -12,9 +12,11 @@ export const DATE_FORMATS = {
     dateTime: 'dd/MM/yyyy, hh:mm:ss aa',
     day: 'dd',
     dayMonthFull: "dd MMMM",
-    timeForEvent: "hh:mm aa"
+    timeForEvent: "hh:mm aa",
+    date: "yyyy-MM-dd"
 }
 
+export const MONTHLY_PLAN = 200000
```

Comment:
czy monthly plan w przyszłości możnaby ustawiać w nowej tabeli w bazie monthConfig? Taki kierownik jak otwierałby nowy miesiąc to by mógł to zmieniać